x/sequencer/keeper/msg_server_decrease_bond_test.go

```diff
@@ -0,0 +1,128 @@
+package keeper_test
+
+import (
+	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dymensionxyz/dymension/v3/x/sequencer/types"
+)
+
+func (suite *SequencerTestSuite) TestDecreaseBond() {
+	suite.SetupTest()
```

Comment:
minor thing, this test could have been structured so that each test case provides args for the creation of the sequencer that the test case is testing against.
Basically, at each test case iteration, do 

```
suite.SetupTest()
suite.App.SequencerKeeper.SetSequencer(tc.sequencer)

resp, err := suite.msgServer.DecreaseBond(suite.Ctx, &tc.msg)
...
```

and the test cases could be like:

```
{
 	name: "sequencer is not bonded",
	sequencer: types.Sequencer{Address: addr, Status: types.Unbonded},
 	msg: types.MsgDecreaseBond{
 		Creator: 	 addr,
	 	DecreaseAmount:  sdk.NewInt64Coin(bondDenom, 10),
 	},
 	expectedErr: types.ErrInvalidSequencerStatus,
},
```

Reply:
@spoo-bar